Fix Copy to Clipboard

RhysLees opened the pull request:
Currently, it seems that the copy to clipboard is broken.

This PR Removes the Clipboard package and instead implements a more 'vanilla' approach. it has a fallback so people on older browsers can still copy.

marcusmoore commented on resources/assets/js/components/PackageDetailFrame.vue:
```diff
@@ -666,11 +663,50 @@ export default {
+
+        copyTextToClipboard() {
+            if (!navigator.clipboard) {
+                fallbackCopyTextToClipboard();
```
I think this should be `this.fallbackCopyTextToClipboard();`?
```suggestion
                this.fallbackCopyTextToClipboard();
```

marcusmoore commented on resources/assets/js/components/PackageDetailFrame.vue:
```diff
@@ -666,11 +663,50 @@ export default {
-    },
+        },
+
+        fallbackCopyTextToClipboard() {
```
Can you remove the `console.logs` in these two methods please?

marcusmoore reviewed (changes requested):
@RhysLees good spot 😄  I have a couple questions and comments for whenever you have the time.

Thanks for all of the PRs lately! 💪🏾

RhysLees commented on resources/assets/js/components/PackageDetailFrame.vue:
```diff
@@ -666,11 +663,50 @@ export default {
+            textArea.select();
+
+            try {
+                var successful = document.execCommand('copy');
```
It's deprecated but it's because it's the backup copy, if a user is on an older browser then it will use backup if they don't support the modern method.

We probably could just remove the fallback entirely as 99% of deva will be running the latest version of their browser.

RhysLees commented:
@marcusmoore I've simplified and removed the fallback as it's unlikely developers will be using really outdated browsers if you want to take another look

> Thanks for all of the PRs lately! 💪🏾

Not a problem :)

marcusmoore commented:
This surprised me but it looks like Brave actually uses the fallback so the latest doesn't work for me. 

@RhysLees What browser are you using? Including the fallback is probably a good bet but I want to make sure it's just not my browser and extension stack that is causing the issue.

Thanks!

RhysLees commented:
> This surprised me but it looks like Brave actually uses the fallback so the latest doesn't work for me. 
> 
> @RhysLees What browser are you using? Including the fallback is probably a good bet but I want to make sure it's just not my browser and extension stack that is causing the issue.
> 
> Thanks!

I use chrome, tested on Firefox aswell. So I add back fallback then?

marcusmoore commented:
Yes please re-add the fallback and thanks again 😄

RhysLees commented:
> Yes please re-add the fallback and thanks again 😄 

No problem give me 30 min and it will be back by then 😁

RhysLees commented:
@marcusmoore Reverted removing fallback if you want to have another look :)

marcusmoore reviewed (approved):
Looks good! Thanks @RhysLees 😄
